rethinking-css.md

```diff
@@ -30,104 +30,104 @@ Part of the problem is that CSS is so easy, that everyone on the team, regardles
 
-While "thinking in components" (be it with help from a library like React or Vue) is absolutely the sensible approach to structuring and scaffolding your views, if 40 components are `display: flex`, why do we write that in a stylesheet 40 times? It's unnecessarily wasteful.
+While “thinking in components” (be it with help from a library like React or Vue) is absolutely the sensible approach to structuring and scaffolding your views, if 40 components are `display: flex`, why do we write that in a stylesheet 40 times? It’s unnecessarily wasteful.
 
-What if we could define an architecture whereby at some threshold, the size of the final CSS would never increase as the project grows?
+What if we could define an architecture whereby, at some threshold, the size of the final CSS would never increase as the project grows?
 
-Teams without dedicated UI Engineers also often reach for some kind pre-fabricated CSS UI framework, such as Bootstrap or Semantic UI, to get them building and realising UI quickly. While the initial development period is promising (foregoing the fact you're now serving to users 200 colours and 20 components you'll never use), bloat continues to worsen as you encounter the next problem…
+Teams without dedicated UI Engineers also often reach for some kind pre-fabricated CSS UI framework, such as Bootstrap or Semantic UI, to get them building and realising UI quickly. While the initial development period is promising (foregoing the fact you’re now serving to users 200 colours and 20 components you’ll never use), bloat continues to worsen as you encounter the next problem…
 
 ## Specificity wars
 
-Specialist UI Engineers have no issue understanding how CSS inheritance works, and also the 3 pillars of the "Cascading" in CSS: importance, specificity and source order. However, any or all of these can and often trip up less familiar developers.
+Specialist UI Engineers have no issue understanding how CSS inheritance works, nor the three pillars of the CSS “Cascade”: importance, specificity and source order. However, any or all of these can and often do trip up less familiar developers.
 
-As you add new features and styles, do they sometimes conflict with existing ones in surprising ways? Do you hack its specificity to get it to work by adding `!important`, an inline `style` or a more "qualified", nested selector? How does this affect the feature you build in 3 months' time? Do you then have to do the same to combat the last hack? It's a never-ending, losing battle.
+As you add new features and styles do they sometimes conflict with existing ones in surprising ways? Do you hack its specificity to get it to work by adding `!important`, an inline `style` or a more “qualified”, nested selector? How does this affect the feature you build in three months’ time? Do you then have to do the same to combat the last hack? It’s a never-ending, losing battle.
```

Comment:
Is "Spell out integers under 10" a standard thing?

Reply:
I got it from some good editors I've worked with who claimed it was sort of a defacto standard. As a reader, I like it.